### ISSUE TITLE
Error when creating PMML from reg:gamma xgboost model

### ISSUE THREAD

**Issue by rwjmiller** [jpmml/jpmml-xgboost#17]
When setting objective = 'reg:gamma' using the xgboost python API and attempting to export the resulting mode as pmml, I get the following error:

```
Exception in thread "main" java.lang.IllegalArgumentException: reg:gamma
    at org.jpmml.xgboost.Learner.load(Learner.java:85)
        at org.jpmml.xgboost.XGBoostUtil.loadLearner(XGBoostUtil.java:47)
            at org.jpmml.xgboost.Main.run(Main.java:123)
                at org.jpmml.xgboost.Main.main(Main.java:116)
```

I am able to create pmml for other types of xgboost models without any problems.

Please could one of the project maintainers add support for this type of model? I lack the ability to do this myself, unfortunately.

**Comment by rwjmiller** [comment]
Thanks for implementing this!

**Comment by vruusmann** [comment]
The `reg:gamma` objective function is only convertible using the JPMML-XGBoost command-line application. If you want to access this functionality from within the `sklearn2pmml` package, then you need to update it manually:
1) Locate your `$PYTHON_LIBRARIES_DIR/site-packages/sklearn2pmml/resources` directory.
2) Delete the `jpmml-xgboost-1.2.2.jar` file from there.
3) Download the `jpmml-xgboost-1.2.3.jar` file from the Maven Central repository (http://search.maven.org/remotecontent?filepath=org/jpmml/jpmml-xgboost/1.2.3/jpmml-xgboost-1.2.3.jar), and put it there.

**Comment by vruusmann** [comment]
The updated JPMML-XGBoost library is included in newly released JPMML-SkLearn version 1.3.10 and SkLearn2PMML version 0.24.2.
